engine/flink/management/dev-model/src/main/resources/defaultModelConfig.conf

```diff
@@ -58,8 +58,7 @@ components {
   }
 }
 
-
-scenarioPropertiesConfig {
+propertiesConfig {
```

Comment:
The previous name was good, we don't need to change it. The lone "properties" may mean many things, "scenario properties" are more explicit

Reply:
ok, i have changed the naming